config.js

```diff
@@ -70,5 +70,17 @@ config.cookie = {
 config.servers = {
     'adminHostname': 'admin.grasshopper.com',
     'adminPort': 2000,
-    'appsPort': 2001
+    'appsPort': 2001,
+    'shibbolethSPHost': 'shib-sp.grasshopper.com'
+};
+
+/**
+ * `config.signing`
+ *
+ * Configuration namespace for the signing logic
+ *
+ * @param  {String}    key     This key will be used to sign sensitive information. It's vital to the security of the system that you change this in production
```

Comment:
We currently don't document the other configuration parameters in this file. Given that someone else might be running this, perhaps it would make sense to do so?